packages/shopware-6-client/__tests__/services/CustomerService/getCustomer.spec.ts

```diff
@@ -18,19 +18,36 @@ describe("CustomerService - getCustomer", () => {
     });
   });
 
-  it("should an current customer's data - using correct token", async () => {
+  it("should return current customer's data - using correct token", async () => {
     mockedAxios.get.mockResolvedValueOnce({
       data: { data: { id: "c370eb5cd1df4d4dbcc78f055b693e79" } }
     });
-    const result = await getCustomer();
+    const result: any = await getCustomer();
     expect(mockedAxios.get).toBeCalledTimes(1);
     expect(mockedAxios.get).toBeCalledWith(getCustomerEndpoint());
+    expect(result).not.toBeNull();
     expect(result.id).toEqual("c370eb5cd1df4d4dbcc78f055b693e79");
   });
 
-  it("should an current customer's data - using incorrect token", async () => {
-    mockedAxios.get.mockRejectedValue(new Error());
-    expect(getCustomer()).rejects.toThrow();
+  it("should return null when user not logged in", async () => {
+    mockedAxios.get.mockRejectedValueOnce({
+      response: {
+        status: 403
```

Comment:
finally we are using proper http codes, not 500 on everything :+1: you know what I mean :)